src/rules/__tests__/no-page-pause.spec.ts

```diff
@@ -1,14 +1,12 @@
-const { runRuleTester, wrapInTest } = require('../lib/utils/rule-tester');
-const rule = require('../lib/rules/no-page-pause');
+import { runRuleTester, wrapInTest } from '../../utils/rule-tester';
```

Comment:
not a strong opinion on it, but I think it would be nice to have tests inside `/tests` and actual impl source inside `/src` like we had before. At least thats what we do in Playwright usually.

Reply:
The reason I moved away from that was because it introduces more complexity with building the output files since multiple root directories result in a non-flat structure so it would end up like `lib/src/index.js` as the output path.  This can be solved by adding a separate tsconfig file for building so that you can type check src and tests but when building outputs it will ignore the test folder.  Having `__tests__` inside `src` makes it a little easier since a simple glob in the `files` array will ignore the spec files.

Reply:
Okay, so after I thought more about it, I decided to move it back 😄.  It will make it more organized for when we do the migration to `@playwright/test`.